starter-code/index.html

```diff
@@ -3,40 +3,90 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link href="./css/main.css" rel="stylesheet">
 </head>
 <body>
-Premium Discover Help Download
 
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
+<!-- nav bar -->
 
-What’s on Spotify?
+<div class="nav-bar">
+  <div class="logo"><img src="./images/spotify-logo.png" alt="logo"></div>
+<div class="links">
+  <ul>
+    <li>Premium</li>
+    <li>Discover</li> 
+    <li>Help</li> 
+    <li>Download</li>
+  </ul>
+</div>
+</div>
+<div class="clear"></div>
+<!-- music for everyone -->
 
-Millions of Songs
-There are millions of
-songs on Spotify
+<div class="everyone">
+<h1>Music for everyone.</h1>
+<p>Spotify is now free on mobile, tablet and computer.
+Listen to the right music, wherever you are.</p>
+</div>
 
-HD Music
-Listen to music as if you
-were listening live
+<!-- whats on spodify -->
 
-Stream Everywhere
-Stream music on your
+<div class="whats-on">
+
+<h2>What’s on Spotify?</h2>
+
+<div class="flex-container">
+<div>
+<h3>Millions of Songs</h3>
+<p>There are millions of
+songs on Spotify</p>
+</div>
+
+<div>
```

Comment:
There is no need to make a div class without a name. Dont make empty `<div>` without giving a class name.